addOns/ascanrulesAlpha/ascanrulesAlpha.gradle.kts

```diff
@@ -10,6 +10,7 @@ zapAddOn {
 
         dependencies {
             addOns {
+                register("ascanrules")
```

Comment:
We should try remove this dependency, maybe use `commonlib`.

Reply:
Shall I move it around in a separate PR and then rebase this?

Reply:
As discussed in the call I’ll start tackling this.